tests/Extensions/Hosting.Services.Remoting.UnitTests/MockService.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Microsoft.ServiceFabric.Services.Remoting;
+
+namespace Microsoft.Omex.Extensions.Hosting.Services.Remoting.UnitTests
+{
+	internal class MockService : IService
```

Comment:
>MockService [](start = 16, length = 11)

Can we use [https://github.com/loekd/ServiceFabric.Mocks](https://github.com/loekd/ServiceFabric.Mocks)?

Reply:
I was not able to find IService implementation here, I believe it's a class that users implement themself, but I might be mistaken.